.github/workflows/version-bump.yml

```
@@ -29,51 +34,44 @@ jobs:
+            BUMP_TYPE="major"
           else
-            echo "type=patch" >> $GITHUB_OUTPUT
-            echo "Bump type: PATCH (default)"
+            BUMP_TYPE="patch"
           fi
 
+          echo "type=$BUMP_TYPE" >> $GITHUB_OUTPUT
+
+      - name: Output bump type
+        # ISSUE #4: Move echo statement to separate step after bump_type completes
+        run: echo "Bump type: ${{ steps.bump_type.outputs.type }}"
```

actionlint (1.7.11)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)